src/GraphQL.NewtonsoftJson/StringExtensions.cs

```diff
@@ -13,7 +13,10 @@ public static class StringExtensions
         /// <param name="json">A JSON-formatted string.</param>
         /// <returns>Inputs.</returns>
         public static Inputs ToInputs(this string json)
-            => json?.ToDictionary().ToInputs();
```

Comment:
As I remember `?.` operator itself first checks left value for `null` so no need to change this code.

Reply:
I thought that too, hence why I refactored to that from the original. But it seems to short-circuit the entire call chain, so you get `null` back rather than the intended `Inputs.Empty`. That's what the unit test helped pick up.

Reply:
> But it seems to short-circuit the entire call chain

Exactly, I didn’t notice the most important thing. So maybe change to
```c#
  public static Inputs ToInputs(this string json) => json?.ToDictionary().ToInputs() ?? Inputs.Empty;
```
to keep code more compact?

Reply:
not so important anyway